spec/system/admin/announcements_spec.rb

```diff
@@ -63,13 +63,14 @@
       end
 
       expect(page).to have_current_path(admin_announcements_path)
-      expect(page).not_to have_content('Test Message')
+      expect(page).to have_no_content('Test Message')
 
       using_session('learner') do
         sign_in(create(:user))
 
         visit home_path
-        expect(page).not_to have_content('Test Message')
+        expect(page).to have_current_path(home_path)
```

Comment:
There's a handful of offences like this I had to fix; I'm assuming because using a positive selector first means you can be sure the page has actually loaded